codegen/smithy-ruby-codegen-test/integration-specs/auth_spec.rb

```diff
@@ -267,7 +267,7 @@ module Auth
 
     describe '#http_bearer_auth' do
       let(:config_hash) do
-        { http_bearer_identity_resolver: identity_resolver }
+        { http_bearer_provider: identity_resolver }
```

Comment:
Do a generic search for identity_resolver too

Reply:
yea, fixed.  Was only in this test